src/main/java/de/hpi/bpt/argos/parsing/EventTypeParser.java

```diff
@@ -0,0 +1,12 @@
+package de.hpi.bpt.argos.parsing;
+
+/**
+ * This interface offers methods to parse eventTypes from .json files.
+ */
+public interface EventTypeParser {
```

Comment:
![MAJOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-major.png 'Severity: MAJOR') Annotate the "EventTypeParser" interface with the @FunctionalInterface annotation [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](http://bpt-lab.org/sonarqube/coding_rules#rule_key=squid%3AS1609)